### PR TITLE
drop md configurations

### PR DESCRIPTION
**Pull request by chenxsan**
We've migrated all md files to mdx in https://github.com/webpack/webpack.js.org/pull/5233 and https://github.com/webpack/webpack.js.org/pull/5298, it's time to drop all those md related configurations.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,8 +121,7 @@
     "remark-extract-anchors": "1.1.1",
     "remark-frontmatter": "^3.0.0",
     "remark-gfm": "^1.0.0",
-    "remark-html": "^13.0.1",
-    "remark-loader": "^4.0.0",
+    "remark-html": "^13.0.0",
     "remark-refractor": "git://github.com/montogeek/remark-refractor.git",
```
Comment (JounQin) [inline package.json:125]: `git` protocol is incorrect for other contributors.

```suggestion
    "remark-refractor": "montogeek/remark-refractor",
```

File: package.json [hunk 2]
```diff
@@ -123,8 +121,7 @@
     "remark-extract-anchors": "1.1.1",
     "remark-frontmatter": "^3.0.0",
     "remark-gfm": "^1.0.0",
-    "remark-html": "^13.0.1",
-    "remark-loader": "^4.0.0",
+    "remark-html": "^13.0.0",
```
Comment (JounQin) [inline package.json:124]: Any issue after removing it?

Comment (chenxsan) [reply package.json:124]: Nope. We have some remark plugins using it for testing, see https://github.com/webpack/webpack.js.org/blob/0f40d4ed63077ec53426bd349f432fe60225a6a9/src/remark-plugins/remark-custom-asides/index.test.js#L10. That's why I added it back.

File: package.json (hunk 1 above)
Comment (chenxsan) [reply package.json:125]: Thanks, I didn't know it works in this way.

**Pull request merged**